linc_convert/modalities/lsm/mosaic.py

```diff
@@ -215,27 +215,16 @@ def convert(
 
     # Write OME-Zarr multiscale metadata
     print("Write metadata")
-    omz.write_ome_metadata(["c", "z", "y", "x"], list(map(float, reversed(voxel_size))))
-
-    if not zarr_config.nii:
-        print("done.")
-        return
-
-    # Write NIfTI-Zarr header
-    # NOTE: we use nifti2 because dimensions typically do not fit in a short
-    # TODO: we do not write the json zattrs, but it should be added in
-    #       once the nifti-zarr package is released
-    shape = list(reversed(omz["0"].shape))
-    shape = shape[:3] + [1] + shape[3:]  # insert time dimension
-    affine = orientation_to_affine(orientation, *voxel_size)
-    if center:
-        affine = center_affine(affine, shape[:3])
-    header = nib.Nifti2Header()
-    header.set_data_shape(shape)
-    header.set_data_dtype(omz["0"].dtype)
-    header.set_qform(affine)
-    header.set_sform(affine)
-    header.set_xyzt_units(nib.nifti1.unit_codes.code["micron"])
-    header.structarr["magic"] = b"nz2\0"
-    omz.write_nifti_header(header)
-    print("done.")
+    voxel_size = list(map(float, reversed(voxel_size)))
+    omz.write_ome_metadata(["c", "z", "y", "x"], list(map(float, voxel_size)))
```

Comment:
Since `voxel_size` elements are already converted to float in the statement above.

```suggestion
    omz.write_ome_metadata(["c", "z", "y", "x"], voxel_size)
```